storage-access.bs

```diff
@@ -155,25 +158,34 @@ When invoked on {{Document}} |doc|, the <dfn export method for=Document><code>ha
 1. If |global| is not a [=secure context=], then [=/resolve=] |p| with false and return |p|.
 1. If |doc|'s [=Document/browsing context=] is a [=top-level browsing context=], [=/resolve=] |p| with true and return |p|.
 1. If the [=top-level origin=] of |doc|'s [=relevant settings object=] is an [=opaque origin=], [=/resolve=] |p| with false and return |p|.
-1. If |doc|'s [=Document/origin=] is [=same origin=] with the [=top-level origin=] of |doc|'s [=relevant settings object=], [=/resolve=] |p| with true and return |p|.
+1. If |doc|'s [=Document/browsing context=] is same authority with |doc|'s [=Document/browsing context=]'s [=top-level browsing context=], [=/resolve=] |p| with true and return |p|.
```

Comment:
Hm, okay, I would have imagined this to operate on BC as I see Document as a concept that doesn't necessarily cross the process boundary, but this will probably change based on where we arrive later either way :)

Reply:
A browsing context doesn't have authority. It's a sequence of documents of many different origins after all. And yeah, you'd need some kind of proxy in an actual implementation.

Reply:
Yeah, that makes sense, thanks!